src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -956,6 +956,26 @@ They are available in two places: `beforeSend`/`beforeBreadcrumb` and `eventProc
 
 #### Hints for Events
 
+##### Use-Case: `before-send`
+
+The `before-send` callback is passed the event and a second argument `hint` which holds one or more hints. Typically this hint holds the original exception so that additional data can be extracted or grouping is affected. `before-send` will be called right before the event is sent to the server, so it’s the last place where you can edit its data.
+
+In this example, the fingerprint is forced to a common value if an exception of a certain type has been caught:
+
+```javascript
+import * as Sentry from '@sentry/browser';
+
+init({
+  beforeSend(event, hint) {
```

Comment:
i think we call this 'hints' elsewhere? if so we should match that in the code sample (I might be wrong though)

Reply:
Everywhere I've seen the parameter used, it's been called `hint`. I think the heading **Hints** is plural only because calling it **Hint** would be weird.

Reply:
It's singular in JS and Go as well.